.github/workflows/container-image.yml

```diff
@@ -61,6 +86,70 @@ jobs:
       with:
         image: "samba-ad-server:latest"
         retention_days: 1
+  build--centos-client:
+    runs-on: ubuntu-latest
+    env:
+      BUILDAH_FORMAT: oci
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build the centos client image
+      run: make OS_NAME=centos build-client
+    # Here we upload samba-client image to artifacts locally for consumption
+    # during samba-toolbox build process.
+    - name: Upload client image
+      uses: ishworkh/docker-image-artifact-upload@v1
+      with:
+        image: "quay.io/samba.org/samba-client:icentos-latest"
```

Comment:
```suggestion
        image: "quay.io/samba.org/samba-client:centos-latest"
```

Reply:
Thanks, I had already found and fixed that error yesterday.